src/TroykaTextLCD.h

```diff
@@ -2,141 +2,143 @@
-#define LCD_OSC_122HZ           0x00
-#define LCD_OSC_131HZ           0x01
-#define LCD_OSC_144HZ           0x02
-#define LCD_OSC_161HZ           0x03
-#define LCD_OSC_183HZ           0x04
-#define LCD_OSC_221HZ           0x05
-#define LCD_OSC_274HZ           0x06
-#define LCD_OSC_347HZ           0x07
+#define LCD_BOOST_OFF 0x00
+#define LCD_OSC_122HZ 0x00
+#define LCD_OSC_131HZ 0x01
+#define LCD_OSC_144HZ 0x02
+#define LCD_OSC_161HZ 0x03
+#define LCD_OSC_183HZ 0x04
+#define LCD_OSC_221HZ 0x05
+#define LCD_OSC_274HZ 0x06
+#define LCD_OSC_347HZ 0x07
 
 // flags Follower control
-#define LCD_FOLLOWER_ON         0x08
-#define LCD_FOLLOWER_OFF        0x00
-#define LCD_RAB_1_00            0x00
-#define LCD_RAB_1_25            0x01
-#define LCD_RAB_1_50            0x02
-#define LCD_RAB_1_80            0x03
-#define LCD_RAB_2_00            0x04
-#define LCD_RAB_2_50            0x05
-#define LCD_RAB_3_00            0x06
-#define LCD_RAB_3_75            0x07
+#define LCD_FOLLOWER_ON 0x08
+#define LCD_FOLLOWER_OFF 0x00
+#define LCD_RAB_1_00 0x00
+#define LCD_RAB_1_25 0x01
+#define LCD_RAB_1_50 0x02
+#define LCD_RAB_1_80 0x03
+#define LCD_RAB_2_00 0x04
+#define LCD_RAB_2_50 0x05
+#define LCD_RAB_3_00 0x06
+#define LCD_RAB_3_75 0x07
 
 class TroykaTextLCD : public Print {
 public:
-    TroykaTextLCD (uint8_t pinBacklight = DEFAULT_PIN_BACKLIGHT, uint8_t i2cAddress = DEFAULT_LCD_I2C_ADDR, TwoWire* wire = &Wire);
-    void begin(uint8_t cols = 16, uint8_t rows = 2, uint8_t charSize = 1);
-    void setContrast(uint8_t contrast);
-    void setBrightness(uint8_t brightness);
-    void setIcon(uint8_t address, uint8_t bit);
-    void clear();
-    void home();
-    void noDisplay();
-    void display();
-    void noBlink();
-    void blink();
-    void noCursor();
-    void cursor();
-    void scrollDisplayLeft();
-    void scrollDisplayRight();
-    void leftToRight();
-    void rightToLeft();
-    void autoscroll();
-    void noAutoscroll();
-    void createChar(uint8_t location, uint8_t charMap[]);
-    void setCursor(uint8_t col, uint8_t row);
-    virtual size_t write(uint8_t value);
-    void command(uint8_t value);
+  TroykaTextLCD(TwoWire *wire = &Wire,
+                uint8_t i2cAddress = DEFAULT_LCD_I2C_ADDR,
+                uint8_t pinBacklight = DEFAULT_PIN_BACKLIGHT);
+  void begin(uint8_t cols = 16, uint8_t rows = 2, uint8_t charSize = 1);
+  void setContrast(uint8_t contrast);
+  void setBrightness(uint16_t brightness);
+  void setIcon(uint8_t address, uint8_t bit);
+  void clear();
+  void home();
+  void noDisplay();
+  void display();
+  void noBlink();
+  void blink();
+  void noCursor();
+  void cursor();
+  void scrollDisplayLeft();
+  void scrollDisplayRight();
+  void leftToRight();
+  void rightToLeft();
+  void autoscroll();
+  void noAutoscroll();
+  void createChar(uint8_t location, uint8_t charMap[]);
+  void setCursor(uint8_t col, uint8_t row);
+  virtual size_t write(uint8_t value);
+  void command(uint8_t value);
```

Comment:
why the command is not private?

Reply:
... and why write is virtual? Who can reload it?

Reply:
override to method:
`size_t write(uint8_t value) override;`

Reply:
[C.128: Virtual functions should specify exactly one of virtual, override, or final.](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c128-virtual-functions-should-specify-exactly-one-of-virtual-override-or-final)

Reply:
OK